pennylane/resources/first_quantization.py

```diff
@@ -68,17 +66,17 @@ class FirstQuantization(Operation):
             import pennylane as qml
             from pennylane import numpy as np
 
-            n = 10000
+            n = 100000
             eta = 156
             omega = 1145.166
 
-            algo = DoubleFactorization(n, eta, omega)
+            algo = FirstQuantization(n, eta, omega)
 
         >>> algo.lamb  # the 1-Norm of the Hamiltonian
         281053.7561435654
```

Comment:
If we change `n`, these numbers should change too, right? Let's double-check that the outputs we give here are the same that a user would observe when typing the same code